tests/integration/test_rids/test_rids_conf.py

```diff
@@ -66,6 +120,32 @@ def set_internal_options_conf(param, value):
 
 
 def test_rids_conf(get_configuration, configure_environment):
+    '''
+    description: Check that RIDS configuration works as expected for the following fields, `remoted.verify_msg_id` and
+                 `remoted.worker_pool`. To do this, it modifies the local internal options with the test case metadata
+                 and restarts Wazuh to verify that the daemon starts or not. Finally, when a correct configuration has
+                 been tested, it restores the `internal_options.conf` as it was before running the test.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configuration from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing
+
+    assertions:
+        - Verify that the RIDS configuration is applied correctly(or not)
```

Comment:
I would add a dot (.) at the end of the brief description.
```suggestion
    assertions:
        - Verify that the RIDS configuration is applied correctly(or not).
```

Reply:
https://github.com/wazuh/wazuh-qa/commit/93ae950e1f55e84b0ad12f9ef4dc99cd7e36b493 solves this